contents/handbook/values.md

```diff
@@ -6,54 +6,60 @@ showTitle: true
+## Make it public
 
-We should never stop iterating. You learn faster and help PostHog perform better by getting what you're working on into the real world. We expect you to ship new designs, features, or whatever is needed for your role in tiny chunks, frequently, and often before you feel ready.
+We're very transparent for so many reasons. 
 
-## 3. Everyone codes
+Internally, a culture of transparency looks like managers telling you to raise feedback directly with the person it concerns instead of solving problems for you, it means changing teams around in public slack channels
 
-... although this doesn't mean everyone has to be a software developer, and not everyone needs experience in this before they join. Our platform's adoption starts with developers using our open-source project, so we use GitHub to share most of our work publicly and to build a large community of technical users.
+Being transparent externally helps us achieve our mission - we write about what we're working on so the world can take advantage of the lessons we're learning, and so they know how to work with us better. Knowing that thousands of people will read our handbook pages forces clearer thinking. And, for free, we can build trust in a way other vendors just choose not to.
 
-No matter your role, being able to use the basics of GitHub helps you understand our audience. Beyond that, we'll encourage you to build your technical skill, rather than delegating more challenging tasks to others, so you become a more effective contributor.
+## Do more weird
 
-## 4. Trust and feedback over process
+So much about how we work is different.
 
-There are two ways to scale – trust and feedback, or process. We choose the former because we're building a wide platform with many products, so autonomy is more important than control. We hire people that work well with high level direction and will step on toes if needed to get things done.
+Weirdness can just be the absurd lengths we are willing to go to. It can mean redesigning an already world-class website, for the 5th time. It can mean shipping _literally_ every product that relates to customer data, with teams of just one to five people competing with $200bn+ companies, successfully.
 
-When giving or receiving feedback, we assume positive intentions and focus on giving specific examples. Many of our team's peak experiences at PostHog have been receiving direct feedback. Feedback should be acknowledged, but what you do with it is up to you - no one built anything great by committee.
+We aren't weird for the sake of it. We want the company perfectly optimized for our strategy. We have small teams, very few others do - because we are going to build 50+ products. We post billboards of our founders' face because no one else is brave enough thus it stands out. Even the little things - like having _pricing_ on our pricing page!
```

Comment:
```suggestion
We aren't weird for the sake of it. We want the company perfectly optimized for our strategy. We have small teams when very few others do, because we are going to build 50+ products. We post billboards of our founders' face because no one else is brave enough thus it stands out. Even the little things - like having _pricing_ on our pricing page!
```